api/grpcserver/activation_service.go

```diff
@@ -42,12 +45,28 @@ func (s *activationService) Get(ctx context.Context, request *pb.GetRequest) (*p
 	atxId := types.ATXID(types.BytesToHash(request.Id))
 	atx, err := s.atxProvider.GetFullAtx(atxId)
 	if err != nil || atx == nil {
-		s.logger.With().Debug("failed to get the ATX", log.Err(err), log.Stringer("id", atxId))
+		s.logger.With().Error("failed to get ATX",
+			log.Stringer("atx id", atxId),
+			log.Err(err),
+		)
 		return nil, status.Error(codes.NotFound, "id was not found")
 	}
-	return &pb.GetResponse{
+	proof, err := s.atxProvider.GetMalfeasanceProof(atx.SmesherID)
+	if err != nil && !errors.Is(err, sql.ErrNotFound) {
+		s.logger.With().Error("failed to get malfeasance proof",
+			log.Stringer("smesher", atx.SmesherID),
+			log.Stringer("id", atxId),
+			log.Err(err),
+		)
+		return nil, status.Error(codes.NotFound, "id was not found")
+	}
+	resp := &pb.GetResponse{
 		Atx: convertActivation(atx),
-	}, nil
+	}
+	if proof != nil {
+		resp.MalfeasanceProof = events.ToMalfeasancePB(atx.SmesherID, proof, false)
```

Comment:
Having `ToMalfeasancePB` in the `events` package feels wrong; I wouldn't think of looking for this function there if I would need it, yet I am also not sure where else to put it.

Reply:
sorry. i know. but it's better for grpcserver -> events than the other way around. 
events pkg already deals with api's pb messages.